src/bp-forums/groups.php

```diff
@@ -270,6 +270,28 @@ public function edit_screen( $group = false ) {
 				$forum_id = (int) is_array( $forum_ids ) ? $forum_ids[0] : $forum_ids;
 			}
 
+			// Get forums that is already associated with group
```

Comment:
@kent-carlo  Can we create a function so we can use that in multiple places? 
We should always create a function if the same code is used in more than one place.